core/tests/protractor_utils/forms.js

```diff
@@ -194,7 +194,10 @@ var RichTextEditor = async function(elem) {
   var closeRteComponentButtonLocator = by.css(
     '.e2e-test-close-rich-text-component-editor');
   // Set focus in the RTE.
-  await waitFor.elementToBeClickable(rteElements.first());
+  await waitFor.elementToBeClickable(
+    rteElements.first(),
+    'First RTE element taking too long to become clickable.'
+  );
   await (await rteElements.first()).click();
```

Comment:
Why are we not using `action.click` to petorm this?

Reply:
Just because we haven't migrated `forms.js` to use `action` functions yet